automation/vars/uni02beta-adoption.yaml

```diff
@@ -0,0 +1,52 @@
+---
+vas:
+  uni02beta-adoption:
+    stages:
+      - path: examples/dt/uni02beta/control-plane/networking/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=5m
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni02beta/control-plane/networking
+        wait_conditions:
+          - >-
+            oc -n metallb-system wait pod
+            -l app=metallb -l component=speaker
+            --for condition=Ready
+            --timeout=5m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+
```

Comment:
The rest of the file is not used for the adoption automation.

Reply:
Deleted the rest. Thanks.